contrib/init/bitcoind.service

```diff
@@ -11,6 +11,7 @@
 
 [Unit]
 Description=Bitcoin daemon
+Documentation=https://github.com/bitcoin/bitcoin/blob/master/doc/init.md
```

Comment:
For other reviewers: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Documentation=

Reply:
https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/#cutthecraphowdoimakesurethatmyservicestartsafterthenetworkisreallyonline :p